packages/ui-pages/src/Pages/__tests__/Pages.test.js

```diff
@@ -77,7 +77,7 @@ describe('<Pages />', async () => {
       </Pages>
     )
 
-    expect(pageSpy).to.have.been.calledOnce()
+    expect(pageSpy).to.have.been.calledTwice()
```

Comment:
Its called twice because we call now `this.props.makeStyles()` in `componentDidMount()`

Reply:
Do you think we should put this as a comment above this test in the code too? Just so the future generations know the reason too :)

Reply:
done